ruby/ql/test/library-tests/modules/calls.rb

```diff
@@ -126,13 +126,13 @@ def funny
 1.bit_length
 1.abs
 
-["a","b","c"].foreach { |i, v| puts "#{i} -> #{v.capitalize}"} # TODO should resolve to String.capitalize
+["a","b","c"].foreach { |i, v| puts "#{i} -> #{v.capitalize}"}
```

Comment:
Great that this is fixed, why is that?

Reply:
Turns out it was fixed for the wrong reasons. The summary seems to depend on having store-load steps so I'll revisit it when adding those (which are relatively simple in type-tracking but more complex in API graphs, so I'd rather not handle it in this PR).